src/content/docs/how-to-work-on-labs.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: How to work on labs
+sidebar:
+  label: Work on Labs
+---
+
+Labs are a type of challenge that presents an empty or almost empty editor to the camper, and a list of user stories to satisfy.
+
+Lab descriptions should have this shape:
+
+```md
+[introduction]
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab.
+
+**User Stories:**
+
+[numbered list with all the details needed to pass the lab]
+
+Note/Hint: (this is optional)
+```
+
+These challenges are used to allow for practice and reinforce concepts and practice problem solving.
+
+It can be that in the user stories there is written something that is not tested (for various reasons, including test limitations), but everything tested must be written in the user stories.
```

Comment:
I would also emphasize that lab demo project can be more involved than the list of user stories presented to campers.